models/articles.models.js

```diff
@@ -1,10 +1,24 @@
 const db = require("../db/connection.js");
 
+exports.selectArticles = () => {
+  return db
+    .query(
+      `SELECT a.author, a.title, a.article_id, a.topic, a.created_at, a.votes, a.article_img_url, 
+      COUNT(c.article_id) as comment_count 
+      FROM articles AS a LEFT OUTER JOIN comments AS c ON c.article_id = a.article_id
+      GROUP BY a.author, a.title, a.article_id, a.topic, a.created_at, a.votes, a.article_img_url
+      ORDER BY a.created_at DESC;`
+    )
```

Comment:
nice psql command! LOVE when people use the shorthands 'a' and 'c' instead of writing it all out. 🎉